### PR TITLE
Enable languages

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: apps/mobile-wallet/src/features/localization/useLocalization.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 
 import 'dayjs/locale/fr'
 import 'dayjs/locale/el'
+import 'dayjs/locale/vi'
```
Comment (mvaivre) [inline apps/mobile-wallet/src/features/localization/useLocalization.ts:21]: Should Greek and German also be here?

File: apps/mobile-wallet/src/features/localization/i18n.ts [hunk 2]
```diff
@@ -22,13 +22,15 @@ import { initReactI18next } from 'react-i18next'
...
+import vi from '../../../locales/vi-VN/translation.json'
 
 i18next.use(initReactI18next).init({
   compatibilityJSON: 'v3',
   resources: {
     'en-US': { translation: en },
     'fr-FR': { translation: fr },
-    'el-GR': { translation: gr }
+    'el-GR': { translation: gr },
+    'vi-VN': { translation: vi }
```
Comment (mvaivre) [inline apps/mobile-wallet/src/features/localization/i18n.ts:33]: Should German also be here?

File: apps/mobile-wallet/src/features/localization/languages.ts [hunk 3]
```diff
@@ -16,10 +16,11 @@ You should have received a copy of the GNU Lesser General Public License
...
 
-export type Language = 'en-US' | 'fr-FR' | 'el-GR'
+export type Language = 'en-US' | 'fr-FR' | 'el-GR' | 'vi-VN'
 
 export const languageOptions: { label: string; value: Language }[] = [
   { label: 'English', value: 'en-US' },
   { label: 'Français', value: 'fr-FR' },
-  { label: 'Ελληνικά', value: 'el-GR' }
+  { label: 'Ελληνικά', value: 'el-GR' },
+  { label: 'Tiếng Việt', value: 'vi-VN' }
```
Comment (mvaivre) [inline apps/mobile-wallet/src/features/localization/languages.ts:25]: Should German also be here?

File: apps/mobile-wallet/src/features/localization/useLocalization.ts (hunk 1 above)
Comment (nop33) [reply apps/mobile-wallet/src/features/localization/useLocalization.ts:21]: Greek is there, the second row (`el`). We don't have full German translations yet, it's very incomplete, so I'd rather wait until we have full coverage before enabling it.

File: apps/mobile-wallet/src/features/localization/i18n.ts (hunk 2 above)
Comment (nop33) [reply apps/mobile-wallet/src/features/localization/i18n.ts:33]: Same answer as before

File: apps/mobile-wallet/src/features/localization/languages.ts (hunk 3 above)
Comment (nop33) [reply apps/mobile-wallet/src/features/localization/languages.ts:25]: aaaand again :)

**Review by mvaivre — APPROVED** [review]
Alrighty

**Pull request merged**